src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -592,6 +593,73 @@ def validator(cmd, namespace):
     return validator
 
 
+def get_content_setting_validator_track2(settings_class, update, guess_from_file=None, process_md5=False):
```

Comment:
We can add `process_md=False` to the original func `get_content_setting_validator()` to reduce deplication